test/functional/timer_sessions_controller_test.rb

```diff
@@ -45,6 +45,13 @@ def setup
     assert response.body.include?(@timer_session.splittable_hours.round(2).to_s)
   end
 
+  test 'index with different server time zone' do
+    @user.preference.update(time_zone: 'Tijuana')
+    get(:index)
+    assert_response 200
+    assert response.body.include?('-8')
```

Comment:
That's a bit minimalistic, no?

Reply:
@schmijos I thought I'll write a spec that is influenced by only the feature I want to test.

Reply:
Agreed. Checking for `assert response.body.include?('data-timer-timezone-value="-8"')` makes more sense